backend/backend.py

```diff
@@ -32,6 +33,20 @@ def __init__(self, width: int, height: int, publish_recording_status_func: Publi
         # Time between recording statuses, in seconds
         self.recording_status_interval = 60
 
+        # time that we last called self.send_camera_is_online
```

Comment:
nit: Capital letter for comments!

Reply:
Done
f2bf442

Reply:
Doesn't seem to be done on this specific line

Reply:
@hallgchris 
![image](https://user-images.githubusercontent.com/50545432/89599299-840d7f00-d8a2-11ea-8033-6bedd30af577.png)

Reply:
Ah It was done for an earlier comment. Now actually done

b07949d